app/controllers/declaration/TransportContainerController.scala

```diff
@@ -187,12 +186,6 @@ class TransportContainerController @Inject()(
     updateExportsDeclarationSyncDirect(_.updateContainers(updatedContainers))
 
   private def redirectAfterAdd(mode: Mode, containerId: String)(implicit request: JourneyRequest[AnyContent]) =
-    if (allowSeals)
-      navigator.continueTo(routes.SealController.displaySealSummary(mode, containerId))
-    else
-      navigator.continueTo(routes.TransportContainerController.displayContainerSummary(mode))
-
-  private def allowSeals(implicit request: JourneyRequest[AnyContent]) =
```

Comment:
All dec types allow seals to be added.  So I decided to get rid of this method to avoid future bugs/confusion.